updated the requirements to fix documentation and coverage testing

Jskarie opened the pull request:
Work on cleaning up the Travis CI issues.

Jskarie committed:
updated the requirements to fix documentation and coverage testing

Jskarie committed:
update the django requirements

saulshanabrook commented:
Thank you! Yeah it's been on my todo list to update this for more recent Django for a while... Let me know when you are done pushing here and I will merge.

Jskarie commented:
It may take awhile.  I haven't used docker, so to test it I either need to
keep pushing to git to test the build, or figure out docker.  I'll try to
get it before the new pip version which makes it necessary.
  James

On Thu, Aug 20, 2020 at 10:29 AM Saul Shanabrook <notifications@github.com>
wrote:

> Thank you! Yeah it's been on my todo list to update this for more recent
> Django for a while... Let me know when you are done pushing here and I will
> merge.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/saulshanabrook/django-simpleimages/pull/29#issuecomment-677735652>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABCSSKWDYTSWOH52ZUDUINTSBU6NHANCNFSM4GCFTZLQ>
> .
>


-- 
  Be happy because life is good

Jskarie committed:
updated to support Pillow 10.0.0
Pillow 10 removed Image.ANTIALIAS, and renamed it Image.LANCZOS
moved to use the new format.  Updated the requirements to require
version 2.7.0 of Pillow which adds LANCZOS

Jskarie committed:
updated the versions of the dependencies

I get an install error

django-simpleimages 1.3.4 has requirement Django<1.10,>=1.5, but you'll have django 4.2.14 which is incompatible.

this is my attempt to address it.  I don't see 1.10 referenced anywhere
but travis or the docs.
